### PR TITLE
doc(supptechinfo): Update with new GWE chapter

### PR DESCRIPTION
**Pull request by emorway-usgs**
- New GWE chapter will appear as chapter 10
- A couple of minor clean-ups remain, but ready for review

### CODE REVIEW

File: doc/SuppTechInfo/groundwater-energy-model.tex [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+Once the dispersion tensor is calculated, the CND Package handles dispersion in a way that is directly analogous to the method used in the DSP Package. As in the DSP Package, the XT3D option is the default method for representing dispersion in the CND Package, and XT3D can be turned off by the user in package input.
+
+\subsubsection{Advanced Stress Packages}
+
+The version of the GWE Model described here offers four advanced stress packages, each of which is analogous and functions similarly to an advanced package in the GWT Model: Lake Energy Transport (LKE), Multi-Aquifer Well Energy Transport (MWE), Streamflow Energy Transport (SFE), and Unsaturated Zone Energy Transport (UZE). The GWE advanced stress packages will not be described in detail here except to note differences from the corresponding GWT packages. The primary difference, of course, is that GWE advanced stress packages deal with transport of thermal energy, not solute mass.
```
Comment (langevin-usgs) [inline doc/SuppTechInfo/groundwater-energy-model.tex:115]: Should mention that these packages explicitly simulate heat transport in their features; however, the SSM could be used instead to represent their effects on the subsurface if they have a known temperature.

File: doc/SuppTechInfo/groundwater-energy-model.tex [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+Transport of thermal energy is sometimes simulated using a solute-transport model. In this approach, which takes advantage of the close analogy between the governing equations for thermal-energy transport and solute transport, the solute-transport model is run with values of input parameters that have been scaled to render the governing equation representative of thermal-energy transport \citep{thorne2006, hechtmendez, mazheng2010}. The GWT Model for \mf can be used in this way. However, the new GWE Model offers the following advantages: model input and output in terms of parameters and in units that relate directly to thermal-energy transport; variation of bulk thermal conductivity with changing saturation; and simulation of thermal conduction through the solid matrix, even in dry cells.
```
Comment (langevin-usgs) [inline doc/SuppTechInfo/groundwater-energy-model.tex:4]: Worth mentioning anything about BUY and VSC?

**Review by langevin-usgs — COMMENTED** [review]
This reads very nicely and contains an appropriate level of detail.  I had a few minor thoughts that can be disregarded  or left for a later time, but I just thought I'd mention them.

File: doc/SuppTechInfo/groundwater-energy-model.tex [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
... aquifer; $\rho_w$ and $\rho_s$ are the densities ($M/L^3$) of the water and solid-matrix material, respectively; $C_{pw}$ and $C_{ps}$ are the specific heats ($E/(M \, deg)$) of the water and solid-matrix material, respectively; $T$ is temperature ($deg$); $t$ is time ($T$); $\matr{q}$ is the vector of specific discharge ($L/T$); $\matr{D}_T$ is the second-order tensor of hydrodynamic dispersion coeffiients for thermal energy transport ($L^2/T$); $q'_s$ is the volumetric flow rate per unit volume of aquifer (defined as positive for flow into the aquifer) for fluid sources and sinks ($1/T$), $T_s$ is the temperature of the source or sink fluid ($deg$), $E_s$ is rate of energy loading per unit volume of aquifer ($M/L^3T$), $\gamma_{1w}$ is the zero-order energy decay rate coefficient in the water ($E/L^3T$), and $\gamma_{1s}$ is the zero-order energy decay rate coefficient in the solid ($E/MT$). Note that $\gamma_{1w}$ is defined on a per-volume-of-water basis, whereas $\gamma_{1s}$ is defined on a per-mass-of-solid basis. Note that $\rho_w$, $\rho_s$, $C_{pw}$, and $C_{ps}$ are assumed to remain constant with time, although the solid properties can vary spatially from cell to cell.
```
[the remark this answers is not in the document]
Comment (emorway-usgs) [reply doc/SuppTechInfo/groundwater-energy-model.tex]: Addressed with 1d326c4 in #1819

File: doc/SuppTechInfo/groundwater-energy-model.tex (hunk 2 above)
Comment (emorway-usgs) [reply doc/SuppTechInfo/groundwater-energy-model.tex:4]: Addressed with ab144f6 in #1819

**Pull request merged**